jgiven-core/src/main/java/com/tngtech/jgiven/impl/ScenarioModelBuilder.java

```diff
@@ -496,17 +496,19 @@ private void addTags(Annotation... annotations) {
         }
     }
 
-    private void addTags(List<Tag> tags) {
+    private void addTags(ResolvedTags tags) {
         if (tags.isEmpty()) {
             return;
         }
 
         if (reportModel != null) {
-            this.reportModel.addTags(tags);
+            this.reportModel.addTags(tags.getDeclaredTags());
+            //The report model needs to declare the parent tags in a tag map, or the tags cannot be displayed.
+            this.reportModel.addTags(tags.getParents());
```

Comment:
maybe we could tryto pass on resolved tags to the report model and the scenario model...